src/main/proto/command_server.proto

```diff
@@ -183,15 +322,25 @@ message File {
   build.bazel.remote.execution.v2.Digest digest = 2;
 }
 
-// Full data that can be used to debug cache misses.
+// Full execution data that can be used to debug cache misses.
 message ExecutionData {
+  // All the action input files, including traversed input directories.
   repeated File input_files = 1;
+
+  // All the local files that needed to be uploaded to remote CAS.
   repeated string input_file_cas_misses = 5;
```

Comment:
I feel this is the most important part of the execution data that it should be upgraded to the action metadata. wdyt?

Reply:
Yeah, the only reason not to do it is that it potentially takes too much space, but I'm not sure how much to worry about that. How about we make this change separately?